lib/devise/multi_email/models/authenticatable.rb

```diff
@@ -49,7 +51,7 @@ def email
       def email=(email)
```

Comment:
I think we should rename this method based on configuration as well, otherwise might be confusing. Can use `define_method`.